src/Fortify/Notifications/AccountCreated.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ARKEcosystem\Foundation\Fortify\Notifications;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Notifications\Messages\MailMessage;
+use Illuminate\Notifications\Notification;
+
+/**
+ * @codeCoverageIgnore
```

Comment:
What's that for?

Reply:
Just something that got missed when copied over from msq